package.json

```diff
@@ -32,7 +32,7 @@
   },
   "dependencies": {
     "@lezer/lr": "^1.4.2",
-    "lezer-feel": "^1.2.9",
+    "lezer-feel": "^1.4.0",
```

Comment:
Could upgrade just to `1.3.0` to get the Camunda change, but didn't see any issues with moving to `1.4.0`